### ISSUE TITLE
Problems where the software does not detecting inputs when the game is in fullscreen in Slippi Melee

### ISSUE THREAD

**Issue by cristianstar67** [bkacjios/m-overlay#47]
I noticed that the overlay does not detect inputs when Dolphin is in fullscreen mode, only when it is a window.

 - OS: Windows
 - Dolphin Version Slippi 2.2.3

What I expected to happen was the overlay to detect my inputs while in fullscreen mode

**Comment by bkacjios** [comment]
Is the program minimized by any chance? Are you in a single monitor setup? This may be an issue with either the graphics backend, driver, or just windows itself.

I've noticed on my setup that the overlay will stop updating when minimized, but I've never had a problem with it not working in fullscreen

**Comment by cristianstar67** [comment]
Yes I have a single monitor setup and i always keep it out, not minimized
